replay_analyser.py

```diff
@@ -85,10 +102,15 @@ def team_average_ratings(replays, cache, tank_info=None):
     for battle in replays:
         teams = [[], []]
         std = battle.get('std')
+        skipReplay = None
         for player in std.get('vehicles').values():
             name = player.get('name')
             tank = player.get('vehicleType').split(':', 1)[1]
-            tier = tank_info.get(tank, {}).get('tier')
+            tier = custom_get_tank(tank, tank_info)
+            if tier is None and args.weighted:
+                skipReplay = True
+                break
```

Comment:
If there is missing tank tier data, it's going to be invalidate the results anyway.
breaking also forces players to be skipped